.changes/unreleased/tinymce-TINY-10391-2024-04-26.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: Open link menu not enabled for image with embedded link.
```

To keep the wording consistent with the related change in this release
```suggestion
body: Open link context menu action was not enabled for links on images.
```